_quarto.yml

```diff
@@ -43,9 +43,9 @@ book:
         - tutorial_dataset_7.qmd
     - part: Using outputs of `traits.build`
       chapters:
-        - using_data.qmd
         - austraits_database.qmd
         - austraits_package.qmd
+        - austraits_build_tutorial.qmd
```

Comment:
I think this tutorial is about using data? If so, Let's rename this. Calling it `austraits_build` makes me think it's about building dataset, not using. Why not just AusTraits?

Reply:
I was trying to separate it from `austraits` R package. Happy to call in AusTraits with capitals